### DISCUSSION TITLE
osx openrv build fails, since homebrew openssl@1.1 is now disabled.

### DISCUSSION THREAD

**Discussion by SamRichardsDisney** [AcademySoftwareFoundation/OpenRV#635] [Build Issues]
Homebrew now is refusing to install openssl@1.1 on osx the brew has been disabled. 

Is it better to just wait for qt6?

**Comment by bernie-laberge** [comment]
Hello @SamRichardsDisney ,
I apologize for the delay. I'm investigating this right now.
I'll keep you posted

**Comment by bernie-laberge** [comment]
Hello @SamRichardsDisney,
I used a brand new macOS Intel instance running Sonoma 14.7.1, setup the instance to build OpenRV according to the [macOS instructions](https://aswf-openrv.readthedocs.io/en/latest/build_system/config_macos.html#building-open-rv-on-macos) and I was able to successfully build OpenRV on it. 
But then I realized that OpenRV doesn't require you to install the brew openssl@1.1 dependency: 
https://aswf-openrv.readthedocs.io/en/latest/build_system/config_macos.html#install-tools-and-build-dependencies

How is your OpenRV build breaking exactly ?

**Comment by SamRichardsDisney** [comment]
Its when I build QT5 from scratch on a M2 macbook in order to build a native openRV, that it requires ***@***.******@***.***>  Its really a qt5 issue, so I’m sure once we goto qt6 that problem will go away.

Sam.

From: Bernard Laberge ***@***.***>
Date: Friday, December 13, 2024 at 1:24 AM
To: AcademySoftwareFoundation/OpenRV ***@***.***>
Cc: Richards, Sam ***@***.***>, Mention ***@***.***>
Subject: Re: [AcademySoftwareFoundation/OpenRV] osx openrv build fails, since homebrew ***@***.*** is now disabled. (Discussion #635)

This Message is From an External Sender
Caution: Do not click links or open attachments unless you recognize the sender and know the content is safe.

Hello @SamRichardsDisney<https://github.com/SamRichardsDisney>,
I used a brand new macOS Intel instance running Sonoma 14.7.1, setup the instance to build OpenRV according to the macOS instructions<https://aswf-openrv.readthedocs.io/en/latest/build_system/config_macos.html#building-open-rv-on-macos> and I was able to successfully build OpenRV on it.
But then I realized that OpenRV doesn't require you to install the brew ***@***.*** dependency:
https://aswf-openrv.readthedocs.io/en/latest/build_system/config_macos.html#install-tools-and-build-dependencies

How is your OpenRV build breaking exactly ?

**Reply by bernie-laberge** [reply]
As an alternative, @geffrak recommends building openssl from scratch. He says openssl is easy to build. Maybe it's worth a try because the Qt6 version is not ready for prime time yet. Close but not there yet.

**Comment by SamRichardsDisney** [comment]
I had made a backup of ***@***.******@***.***> so I just used that, which worked for me.
If you build from scratch, can you use a newer version?

Sam.

From: Bernard Laberge ***@***.***>
Date: Friday, December 13, 2024 at 4:30 PM
To: AcademySoftwareFoundation/OpenRV ***@***.***>
Cc: Richards, Sam ***@***.***>, Mention ***@***.***>
Subject: Re: [AcademySoftwareFoundation/OpenRV] osx openrv build fails, since homebrew ***@***.*** is now disabled. (Discussion #635)

This Message is From an External Sender
Caution: Do not click links or open attachments unless you recognize the sender and know the content is safe.

As an alternative, @geffrak<https://github.com/geffrak> recommends building openssl from scratch. He says openssl is easy to build. Maybe it's worth a try because the Qt6 version is not ready for prime time yet. Close but not there yet.

**Reply by bernie-laberge** [reply]
I don't know if the Qt build would be happy with a newer version of OpenSSL
